src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -85,25 +99,14 @@ public static IAEItemStack loadItemStackFromNBT( final NBTTagCompound i )
 			return null;
 		}
 
-		final AEItemStack item = AEItemStack.create( itemstack );
+		final AEItemStack item = AEItemStack.fromItemStack( itemstack );
 		item.setStackSize( i.getLong( "Cnt" ) );
 		item.setCountRequestable( i.getLong( "Req" ) );
 		item.setCraftable( i.getBoolean( "Craft" ) );
 		return item;
 	}
 
-	@Nullable
-	public static AEItemStack create( final ItemStack stack )
-	{
-		if( stack.isEmpty() )
-		{
-			return null;
-		}
-
-		return new AEItemStack( AEItemStackRegistry.getRegisteredStack( stack ), stack.getCount() );
-	}
-
-	public static IAEItemStack loadItemStackFromPacket( final ByteBuf data ) throws IOException
+	public static IAEItemStack fromPacket( final ByteBuf data ) throws IOException
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)